optaplanner-core/src/main/java/org/optaplanner/core/impl/localsearch/decider/acceptor/greatdeluge/GreatDelugeAcceptor.java

```diff
@@ -0,0 +1,115 @@
+package org.optaplanner.core.impl.localsearch.decider.acceptor.greatdeluge;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.localsearch.decider.acceptor.AbstractAcceptor;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchMoveScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchPhaseScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchStepScope;
+import org.optaplanner.core.impl.score.ScoreUtils;
+
+public class GreatDelugeAcceptor extends AbstractAcceptor {
+
+    private Score initialLevel = null;
+
+    private Double rainSpeed = null;
+    private Double rainSpeedRatio = null;
+
+    private int levelsLength = -1;
+    private double[] initialLevelScoreLevels;
+    private double[] initialLevelScoreLevelsNegative;
+    private double[] levelScoreLevels;
+
+    private double levelMinimum = 0;
+    private final double THRESHOLD = .0000001;
+
+    // Good value to come out from
+    private final double DEFAULTRAINSPEEDRATIO = 0.99999995;
```

Comment:
Let's change this to `DEFAULT_RAIN_SPEED_RATIO`.

Reply:
Maybe also use underscores to make it clear how many decimal places are there (`0.999_999_95` or some other grouping style that makes most sense in this case, for example `0.99_999_995` would make sense if it means 99.999,995%).